LibreHardwareMonitorLib/Hardware/Motherboard/SuperIOHardware.cs

```diff
@@ -2829,6 +2829,58 @@ private static void GetNuvotonConfigurationD(ISuperIO superIO, Manufacturer manu
 
                             break;
                         }
+                        case Model.ROG_STRIX_X570_I_GAMING: //NCT6798D
+                        {
+                            v.Add(new Voltage("Vcore", 0, 10, 10));
+                            v.Add(new Voltage("+5V", 1, 4, 1)); //Probably not updating properly
```

Comment:
Do you want to comment these out or leave in?

Reply:
I copied that bit of code from the Asus ROG strix B550i board.  I'm open to recommendations.  From what I can see a value is certainly reported but it is exactly 5V all the time.  I can remove the comment if you prefer.

Reply:
I see, it's OK, let's leave it :)